android-components/components/browser/state/src/test/java/mozilla/components/browser/state/action/TranslationsActionTest.kt

```diff
@@ -531,6 +531,48 @@ class TranslationsActionTest {
         assertTrue(tabState().translationsState.pageSettings?.neverTranslateSite!!)
     }
 
+    @Test
+    fun `WHEN an UpdatePageSettingAction is dispatched for UPDATE_ALWAYS_TRANSLATE_LANGUAGE AND UPDATE_ALWAYS_TRANSLATE_LANGUAGE THEN must be opposites of each other or both must be false `() {
+        // Action started to update the always offer setting to true
```

Comment:
Noticing we don't assert the initial state in some of these tests here. I think in general we should assert the initial state to ensure dispatching the action did indeed yield a change in the state.

Reply:
Updated here! (Didn't update other areas to keep this a small and quick patch. Agree it is a good practice!)